src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java

```diff
@@ -30,63 +35,85 @@
 	 * 			the y coordinate of this tile
 	 * @param type
 	 * 			the type of this tile
-     */
+	 */
 	public MazeTile(int x, int y, TileType type) {
 		this.position = new Vector2f(x, y);
 		this.explored = false;
-
+		this.type = type;
+		
 		switch (type) {
-			case FLOOR:
-				this.height = 0;
-				this.color = ColorRGBA.LightGray;
-				break;
-			case WALL:
-				this.height = 3;
-				this.color = ColorRGBA.Blue;
-				break;
-			case CORRIDOR:
-				this.height = 0;
-				this.color = ColorRGBA.DarkGray;
-				break;
-			default:
-				throw new IllegalArgumentException("Invalid TileType: " + type);
+		case FLOOR:
+			this.height = 0;
+			this.color = ColorRGBA.LightGray;
+			break;
+		case WALL:
+			this.height = 3;
+			this.color = ColorRGBA.Blue;
+			break;
+		case CORRIDOR:
+			this.height = 0;
+			this.color = ColorRGBA.DarkGray;
+			break;
+		default:
+			throw new IllegalArgumentException("Invalid TileType: " + type);
 		}
```

Comment:
I would opt that the original indentation is more correct. I do not know why eclipse by default formats the code without indentation increase for switch.

(However, this is very minor and possibly a personal habit/preference. Either way is fine by me)